source/partials/next_release/_release.md.erb

```diff
@@ -1,35 +1,72 @@
-<h4>Improvements</h4>
+<h4>New dashboard</h4>
+
+  We have made several improvements to the new dashboard the pipeline personalization features.
+
+  **Grouping of pipelines**
+
+  You can now group pipelines by their pipeline groups, or environments
+
+  <figure class='shrink-figure-70'>
+    <%= image_tag 'releases/18.8.0/dashboard-grouping.gif', alt: 'grouping by environment or pipeline group name' %>
+  </figure>
+
+  **Multiple named pipeline views**
+
+  You can create multiple personalizations depending on your preferences
+
+  <figure class='shrink-figure-70'>
+    <%= image_tag 'releases/18.8.0/create-new-view.gif', alt: 'create new personalization view' %>
+  </figure>
+
+  Make it interesting by only showing you what is building or failing
+
+  <figure class='shrink-figure-70'>
+    <%= image_tag 'releases/18.8.0/by-pipeline-status.gif', alt: 'show only building and failed pipelines' %>
+  </figure>
 
-  * <%= link_to_commit '3541529f60044fec297f42efce08ca65d14651ad', 'Upgrade to TFS SDK version 14.118.0.' %>
-  * <%= link_to_commit 'ebdfb7dff06029d03198740701a6ef7e74482a31', 'Do not log server info at shutdown.' %>
+  **Re-order views based on your preference**
 
+  <figure class='shrink-figure-70'>
+    <%= image_tag 'releases/18.8.0/reorder-views.gif', alt: 'reorder views' %>
+  </figure>
 
-<h4>Library upgrades</h4>
+**Accessibility Improvements**
 
-  In order to move towards supporting JRE version 9 and upwards, this release upgrades several libraries used by GoCD, including spring, hibernate, mybatis. See <%= link_to_issue 4785 %>, <%= link_to_issue 4749 %> and <%= link_to_issue 4768 %>.
+We have made some accessibility improvements, which should make it easier for people using screenreaders to use the new dashboard.
+
+<h4>New APIs</h4>
+
+  * <%= link_to_issue 4936, "Introduced an API to configure #{link_to_api 'system-admins', 'GoCD system admin users'}." %>
+
+
+<h4>Improvements</h4>
+
+  * <%= link_to_commit '1ca78a37f992affd6dfd3aa3bae5b75f502c2618', "Improve log messages at server startup when plugins are not configured" %>
+  * <%= link_to_issue '5041', "Pipelines created via the PipelineConfig API shouldn't be created paused by default" %>
```

Comment:
Since its an improvement, what about `Pipelines created via the PipelineConfig API are not paused by default now.`?